internal/appsec/emitter/graphqlsec/result.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package graphqlsec
+
+// Result describes the result of a GraphQL operation.
+type Result struct {
```

Comment:
This type is used in multiple operation results. I suggest to follow the systematic approach we have in httpsec and grpcsec and following this template:

```
type (
 {{ operationName }}Operation struct {...}
 {{ operationName }}OperationArgs struct {...}
 {{ operationName }}OperationRes struct {...}
)
```

My reasoning is to make it simple to navigate from one abstract representation to another and easily find the operation along with their expected arguments and results. For instance, I had to search for all the instances of Result here to find how it was used, whereas a strictly defined one for each operation is simpler to understand how to interact with it.

Reply:
Fixed in 910000ceab8b2e4bed447d589361785c680fd631